website_multi_image_zoom/controllers/main.py

```diff
@@ -10,6 +10,6 @@ class WebsiteSale(main.WebsiteSale):
                 methods=['POST'])
     def get_variant_images(self, **post):
         variant_images = request.env['product.image'].sudo(). \
-            search([('product_variant_id', '=', int(post.get('product_id')))])
+            search([('product_variant_id', 'in', int(post.get('product_id')))])
```

Comment:
the right end of domain is not list, so 'in' is not right.

Reply:
I fixed this bug and also, some other bugs and a little feature update in my latest pull requests. Please review.